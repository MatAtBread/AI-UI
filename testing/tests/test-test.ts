console.log("Index [1]");
console.log(new Date());
console.log([new Date()]);
console.log(1);
console.log("one");
console.log({
    s: "string",
    n: 2,
    d: new Date()
},['one',1]);
console.log("one");
