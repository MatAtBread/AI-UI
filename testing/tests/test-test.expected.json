[
  [1,"Index \\[1\\]"],
  [2,"^\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\d\\.\\d+([+-][0-2]\\d:[0-5]\\d|Z)$"],
  [3,["^\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\d\\.\\d+([+-][0-2]\\d:[0-5]\\d|Z)$"]],
  [4,1],
  [5,"one"],
  [6,{"s":"string","n":2,"d":"^\\d{4}-[01]\\d-[0-3]\\dT[0-2]\\d:[0-5]\\d:[0-5]\\d\\.\\d+([+-][0-2]\\d:[0-5]\\d|Z)$"},["one",1]],
  [7,"one"]
]